packages/yuudachi/src/util/generateHistory.ts

```diff
@@ -136,33 +111,218 @@ export async function generateHistory(
 		...embed,
 	};
 
-	const summary: string[] = [];
-	let truncated = false;
+	// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+	if (cases.length > 0) {
```

Comment:
This is unneeded. If you iterate over an empty array it just won't do anything.

Reply:
Fixed in d514aad